### PR TITLE
add(poetry support)

### PR DESCRIPTION
**Pull request by ChrOertlin**
### Added

adds poetry support closes: #83

### Fixed

**Steps to consider while deploying**
- Configuration changes:
- Documentation updates:
- Inform users by email:

### Review:
- [ ] Code approved by
- [ ] Tests executed on stage by:  (Document the test done with screen shots and description.)
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by ChrOertlin** [comment]
# container can be pulled and build
image

# works on stage
image

File: Dockerfile [hunk 1]
```diff
@@ -20,10 +20,9 @@ WORKDIR /home/worker/app
 COPY . /home/worker/app
 
 # Install app requirements
-RUN pip install -r requirements.txt
-
-# Install app
-RUN pip install -e .
+RUN pip install poetry
+RUN poetry config virtualenvs.create false \
```
Comment (diitaz93) [inline Dockerfile:24]: in cg we use:
```
RUN pip install --no-cache-dir poetry \
    && poetry config virtualenvs.create false \
    && poetry install --no-interaction --no-ansi
```
should we add the flags here as well?

**Review by diitaz93 — APPROVED** [review]
👍

File: Dockerfile (hunk 1 above)
Comment (ChrOertlin) [reply Dockerfile:24]: good call. it works without but thehy seem to add robustness.

**Pull request merged**